src/progress_bar.rs

```diff
@@ -4,6 +4,7 @@ use std::sync::MutexGuard;
 use std::sync::{Arc, Mutex, Weak};
 use std::time::{Duration, Instant};
 use std::{fmt, mem};
+use std::fmt::Display;
```

Comment:
Should import fmt::self here, too, and remove it from the other place.

Reply:
Will do